src/platform/plugins/shared/dashboard/public/dashboard_api/load_dashboard_api/load_dashboard_api.ts

```diff
@@ -73,6 +75,24 @@ export async function loadDashboardApi({
     getDashboardBackupService().storeViewMode(viewMode);
   }
 
+  const getUserAccessControlData = async () => {
```

Comment:
Can you call `getUserAccessControlData` in parallel to `getUserAccessControlData`, so use `Promise.all([loadDashboardState, getUserAccessControlData])`.  Can you also move `getUserAccessControlData` implementation into its own file?